### ISSUE TITLE
Cannot run WIndows build

### ISSUE THREAD

**Issue by sryze** [harelba/q#306]
When I run q.exe:

```
The code execution cannot proceed because python38.dll was not found. Reinstalling the program may fix this problem. 
```

I expected it to run without requiring any DLLs as mentioend on the website:

> q is packaged as a compiled standalone-executable that has no dependencies, not even python itself

Could you look into linking with Python statically in the Windows build?

Thanks.

**Comment by harelba** [comment]
The windows build also uses static linking, so this is very odd. 

Can you send some details about your windows? Perhaps it will provide a clue as to why it's not working.

**Comment by sryze** [comment]
Windows 10 x64

It looks like python38.dll was dynamically linked:

![image](https://user-images.githubusercontent.com/4525736/207400239-a5f297a2-0d28-44d6-860d-153fc90079ef.png)

**Comment by sryze** [comment]
Accoding to https://pyoxidizer.readthedocs.io/en/stable/pyoxidizer_packaging_static_linking.html#building-statically-linked-binaries-on-windows to produce a statically linked .exe on Windows, it's necessary to add `flavor="standalone_static"` to this line in pyoxidizer.bzl:

```
dist = default_python_distribution(python_version=PYTHON_VERSION)
```

I just tried this and it appears to indeed link with Python statically.

![image](https://user-images.githubusercontent.com/4525736/207536518-80f011a1-30e6-4da7-b328-ccd168ee619f.png)

**Comment by harelba** [comment]
yes, i remember now... There was an issue with github actions' Windows support which prevent this from working well.

Perhaps now github actions improved their support for this. I'll take a look. 

Thanks a lot for checking this out.
